pkgs/applications/misc/khal/default.nix

```diff
@@ -90,6 +90,8 @@ python3.pkgs.buildPythonApplication rec {
   disabledTests = [
     # timing based
     "test_etag"
+    "test_bogota"
+    "test_event_no_dst"
```

Comment:
Are those two timing related?

Reply:
yes, they are related to timezones

Reply:
timezones are not timing. Timing is when the tests fail due to high machine load.